### DISCUSSION TITLE
openSUSE MicroOS pulls extremely slowly?

### DISCUSSION THREAD

**Discussion by valkenburg-prevue-ch** [mysticaltech/terraform-hcloud-kube-hetzner#507] [Q&A] [answered]
Hi all,

is it just me, or does pulling openSUSE MicroOS suddenly take up to one hour, on each node, when installing a cluster from scratch? My logs show things like:
```
openSUSE-M   2%  11.93M   145KB/s    eta 42m 36s
```
It starts out at 2MB/s, but then quickly gets throttled down to some 100KB/s.

I am using the default mirror, https://ftp.gwdg.de/pub/opensuse/repositories/devel:/kubic:/images/openSUSE_Tumbleweed/openSUSE-MicroOS.x86_64-OpenStack-Cloud.qcow2 .

This was throttling did not happen up until a week ago.

**Comment by valkenburg-prevue-ch** [comment, marked as answer]
Ok, I rtfm'ed, and switched to another mirror for opensuse_microos_mirror_link, picked from https://download.opensuse.org/tumbleweed/appliances/openSUSE-MicroOS.x86_64-OpenStack-Cloud.qcow2.mirrorlist as per the comments in kube.tf.example.

While doing so, found that the regex which validates that variable needed a fix:
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/508

And I understand that a more fundamental solution to this issue is being worked on at https://github.com/kube-hetzner/packer-hcloud-microos .

**Reply by mysticaltech** [reply]
Just released v1.6.12 with your fix! 🚀 🙏

**Reply by valkenburg-prevue-ch** [reply]
Sweet!
